src/shippingservice/quote.go

```diff
@@ -46,8 +46,5 @@ func CreateQuoteFromFloat(value float64) Quote {
 
 // quoteByCountFloat takes a number of items and generates a price quote represented as a float.
 func quoteByCountFloat(count int) float64 {
```

Comment:
Is this method removable as well? Or is it being used elsewhere other than in `main.go`?

Reply:
Good point, this is also removable, since it is a flat rate. We can just set the value in `CreateQuoteFromCount` directly